README.md

```diff
@@ -30,9 +30,10 @@ Releases can be found on the [release list][releases].
 
 ## System requirements
 
-- A not too ancient Linux, or Windows Vista or later, or OSX 10.8 or later.
-- A somewhat capable CPU. Hardware decoding might sometimes help if the CPU
-  is too slow to decode video realtime, but must be explicitly enabled with
+
+- A not too ancient Linux, Windows Vista or later, or OSX 10.8 or later.
+- A somewhat capable CPU. Hardware decoding might help if the CPU is too
+  slow to decode video in realtime, but must be explicitly enabled with
```

Comment:
These are unneeded.

Reply:
I actually like the new version better. "sometimes" was redundant  and "in realtime" seems more natural than just "realtime".